### ISSUE TITLE
Suggestion: a more explicit error message / gammu doc correction?

### ISSUE THREAD

**Issue by nicolashainaux** [gammu/wammu#52]
Hello,

First of all, thanks for your excellent software!

I had to struggle a lot to discover why my computer wouldn't connect to my mobile phone. The error message in Wammu was saying the serial port was busy, but as I had not started any other software than Wammu, so I was wondering how this was possible. Finally, the "Phone does not support enabled echo, it can not work with Gammu!" led me to the doc and the `fuser` command line that helps. I don't know if there's a way to display this more helpful message in the dialog box of Wammu, instead of only in the logs?

Then, when I executed `fuser -va /dev/ttyACM0`, it just displayed:

```
$ fuser -va /dev/ttyACM0
                     UTIL.       PID ACCÈS  COMMANDE
/dev/ttyACM0:
```

So I thought there was no process and Wammu was just maybe bugging or something, and it is hard to guess that there is actually a process using the device. I don't know why `fuser` does not tell that it may not show processes that the current user has no right on.

Maybe it would be good to tell in gammu's documentation that it may be required to run `fuser` with admin rights, in order to see all processes? I don't know if it applies to any Linux distribution (I use Manjaro). This led me to the solution of the problem:

```
$ sudo fuser -va /dev/ttyACM0
                     UTIL.       PID ACCÈS  COMMANDE
/dev/ttyACM0:        root        337 F.... ModemManager
```

**Comment by nijel** [comment]
Well it's already in the docs: https://wammu.eu/docs/manual/faq/general.html#echo

**Comment by nicolashainaux** [comment]
Not exactly, and I forgot to explicitly mention that my suggestion is about this very paragraph, what did help but also confused me in the same time.

I think it would help to mention, in this paragraph, that `fuser` may silently hide some results if not run with administrative rights (`sudo fuser -va /dev/ttyACM0` returns results while `fuser -va /dev/ttyACM0` returns no results at all in the same time).

For instance, after the command `fuser -va /dev/ttyACM0`, maybe add something like "Caution: you may need to run `fuser` with administrative rights to get any result (for instance with `sudo`)." ?
